src/main/java/seedu/address/logic/parser/RecipeViewCommandParser.java

```diff
@@ -19,20 +17,13 @@ public class RecipeViewCommandParser implements Parser<RecipeViewCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public RecipeViewCommand parse(String args) throws ParseException {
-        String trimmedArgs = args.trim();
-        int uuid;
-        if (trimmedArgs.isEmpty()) {
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, RecipeViewCommand.MESSAGE_USAGE));
-        }
         try {
-            uuid = Integer.parseInt(trimmedArgs);
-        } catch (NumberFormatException e) {
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, RecipeViewCommand.MESSAGE_USAGE));
-        }
-        if (!UniqueId.isValidId(uuid)) {
-            throw new ParseException(String.format(MESSAGE_INVALID_RECIPE_DISPLAYED_INDEX,
-                    RecipeViewCommand.MESSAGE_USAGE));
+            String trimmedArgs = args.trim();
```

Comment:
Just nitpicking, but maybe only place the lines that have the potential to throw errors to be within the try block for easier debugging in the future